trackers/react-native-tracker/src/subject.ts

```diff
@@ -0,0 +1,87 @@
+import { CorePluginConfiguration, PayloadBuilder, TrackerCore } from '@snowplow/tracker-core';
+import { ScreenSize, SubjectConfiguration } from './types';
+
+export function newSubject(core: TrackerCore, configuration?: SubjectConfiguration) {
+  let domainUserId: string | undefined;
+  let networkUserId: string | undefined;
+
+  const addSubjectToPayload = (payload: PayloadBuilder): void => {
```

Comment:
Another style nitpick, there's a return type here which the other functions declared in this scope don't have. Not fussed again which way round you want to do it, as long as it's consistent.